packages/client/auth/package.json

```diff
@@ -15,11 +15,15 @@
     "types": "./dist/index.d.ts",
     "files": ["dist", "src", "LICENSE"],
     "scripts": {
-        "build": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --minify --dts --sourcemap",
-        "dev": "tsup src/index.ts --clean --format esm,cjs --outDir ./dist --dts --sourcemap --watch"
+        "build": "tsup",
+        "dev": "tsup --watch",
+        "test": "vitest run"
     },
     "dependencies": {
+        "@crossmint/client-sdk-base": "workspace:*",
         "@crossmint/common-sdk-auth": "workspace:*",
-        "jwt-decode": "4.0.0"
+        "@crossmint/common-sdk-base": "workspace:*",
+        "jwt-decode": "4.0.0",
+        "@farcaster/auth-kit": "0.6.0"
```

Comment:
Do we have to depend on farcaster on our SDKs?
We should be able to make it so that the developer brings farcaster in if they wish to login with it, right?

Reply:
After reading the entire PR, I think this comment doesn't make much sense from my part.
I do think we should be able to split up auth providers in the future, but for now I think this should be fine.